runtime/lib/ttnn/runtime.cpp

```diff
@@ -9,6 +9,11 @@
 #include "ttmlir/Target/TTNN/Target.h"
 #include "ttmlir/Version.h"
 
+// references to alignment byte values
+constexpr static const std::uint32_t &REF_NOC_L1_ALIGNMENT = L1_ALIGNMENT;
+constexpr static const std::uint32_t &REF_PCIE_ALIGNMENT = PCIE_ALIGNMENT;
+constexpr static const std::uint32_t &REF_NOC_DRAM_ALIGNMENT = DRAM_ALIGNMENT;
```

Comment:
Let's change these to be:
```
constexpr static std::uint32_t kNocL1Alignment = L1_ALIGNMENT;
```

Or just use `L1_ALIGNMENT` and others directly in `CreateChipDesc`.